demo/index.html

```diff
@@ -20,282 +20,279 @@
-      on going into the garden at once; but, alas for poor Alice! when
-      she got to the door, she found she had forgotten the little
-      golden key, and when she went back to the table for it, she found
-      she could not possibly reach it: she could see it quite plainly
-      through the glass, and she tried her best to climb up one of the
-      legs of the table, but it was too slippery; and when she had
-      tired herself out with trying, the poor little thing sat down and
-      cried.</p>
-
-      <p>'Come, there's no use in crying like that!' said Alice to
-      herself, rather sharply; 'I advise you to leave off this minute!'
-      She generally gave herself very good advice, (though she very
-      seldom followed it), and sometimes she scolded herself so
-      severely as to bring tears into her eyes; and once she remembered
-      trying to box her own ears for having cheated herself in a game
-      of croquet she was playing against herself, for this curious
-      child was very fond of pretending to be two people. 'But it's no
-      use now,' thought poor Alice, 'to pretend to be two people! Why,
-      there's hardly enough of me left to make <i>one</i> respectable
-      person!'</p>
-
-      <p>Soon her eye fell on a little glass box that was lying under
-      the table: she opened it, and found in it a very small cake, on
-      which the words 'EAT ME' were beautifully marked in currants.
-      'Well, I'll eat it,' said Alice, 'and if it makes me grow larger,
-      I can reach the key; and if it makes me grow smaller, I can creep
-      under the door; so either way I'll get into the garden, and I
-      don't care which happens!'</p>
-
-      <p>She ate a little bit, and said anxiously to herself, 'Which
-      way? Which way?', holding her hand on the top of her head to feel
-      which way it was growing, and she was quite surprised to find
-      that she remained the same size: to be sure, this generally
-      happens when one eats cake, but Alice had got so much into the
-      way of expecting nothing but out-of-the-way things to happen,
-      that it seemed quite dull and stupid for life to go on in the
-      common way.</p>
-
-      <p>So she set to work, and very soon finished off the cake.</p>
-
-    </paper-dialog-scrollable>
-
-    <div class="footer">
-      Lewis Carroll
-    </div>
-
-  </section>
+<body unresolved class="centered">
+  <h3>
+    <code>paper-dialog-scrollable</code> shows a divider at the top and/or bottom
+    indicating more content, depending on scroll position
+  </h3>
+  <demo-snippet class="centered-demo">
+    <template>
+      <style>
+        .header {
+          padding: 8px 24px;
+        }
+
+        .footer {
+          padding: 8px 24px;
+          text-align: right;
+        }
+
+        paper-dialog-scrollable {
+          height: 300px;
+        }
+      </style>
+
+      <section>
+
+        <div class="header">
```

Comment:
can you make the header and the footer one line (`<div class="header">Alice In Wonderland</div>`, and remove all of the empty lines? this is a hell of a block of code to look at :)

Also, maybe add something like this, too? https://github.com/PolymerElements/paper-card/blob/master/demo/index.html#L42